src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -524,7 +526,13 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
                 return args;
             }
 
-            var newList = VisitList(args);
+            var first = (BoundExpression)Visit(args[0]);
+
+            bool stackNonEmpty = _stackNonEmpty;
+            _stackNonEmpty = true;
```

Comment:
>_stackNonEmpty = true; [](http://example.com/codeflow?start=12&length=22)

This doesn't make me feel any better. What would make me feel better is no changes in this function and the machinery somehow do the right thing.